Add @obensource as CommComm member

bnb opened the pull request:
Initial pass at adding @obensource as a CommComm member. Can add email address once this passes and they approve. As an observer, @obensource has hit the three month attendance period to become a member.

obensource commented:
Wahoo! 🎉🎉🎉

My public email is benpmichel@gmail.com 😎

Thanks again @bnb, stoked! 🙌

bnb commented:
Vote has closed, welcome to the Node.js Community Committee @obensource!

Next steps:
- [ ] Update this PR with your email
- [ ] Add your email to the CommComm alias
- [ ] Add you to the CommComm GitHub team

bnb commented:
I've completed the tasks listed above. Email has been updated, added to the mailing list via PR, invited to the CommComm team, and removed as an observer from the issue creator tool.

@nodejs/community-committee PTAL – this should be good to merge 👍
